Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +15,67 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        public static string Name { get; set; }
```

Comment:
Since we don't delve into OOP now, everything is static in your eyes, which is fine. But here is a little background: static means global- can be accessed from anywhere, is always in memory and there is ever going to be a single value.

In the example here, name, surname and other fields will be assigned for one person at a time only. When they get assigned for a second person, the first person will loose the context.

Therefore they should be kept in local function scope, rather static class scope.

Reply:
Therefore, it's safe to remove all of those fields and move each of them close to where you use them- local function scope.

Minimizing scope is one of the things that a programmer tries to seek.